graylog2-web-interface/src/pages/SidecarEditConfigurationPage.jsx

```diff
@@ -52,6 +52,10 @@ const SidecarEditConfigurationPage = createReactClass({
     return !this.state.configuration || !this.state.configurationSidecars;
   },
 
+  _configurationVariableRenameHandler(oldname, newname) {
+    this.configurationForm._replaceConfigurationVariableName(oldname, newname);
```

Comment:
I see a couple of problems with this:
- This works but does not feel like react. Ideally you want to let `ConfigurationForm` handle its own state, not force a change from outside the component. I think there is a piece of state that should defined in this component, and it's the variable list. In that way you could pass the list down to both `ConfigurationForm` and `ConfigurationHelper`, allowing `ConfigurationForm` to rename variables when they change in its props.
- Here you are using a private method of the configurationForm component (at least that's the convention of using the `_` prefix on a function name).